tests/models/test_neuralgcm.py

```diff
@@ -0,0 +1,154 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import pytest
+
+from torchgeo.models import NeuralGCM_Weights, neuralgcm
+
+
+class TestNeuralGCM:
+    def test_invalid(self) -> None:
+        with pytest.raises(ValueError, match='Please specify a weights enum'):
+            neuralgcm()
+
+    @pytest.mark.slow
+    @pytest.mark.parametrize(
+        'weights',
+        [
+            NeuralGCM_Weights.DETERMINISTIC_0_7_DEG,
+            NeuralGCM_Weights.DETERMINISTIC_1_4_DEG,
+            NeuralGCM_Weights.DETERMINISTIC_2_8_DEG,
+            NeuralGCM_Weights.STOCHASTIC_1_4_DEG,
+            NeuralGCM_Weights.STOCHASTIC_PRECIP_2_8_DEG,
+            NeuralGCM_Weights.STOCHASTIC_EVAP_2_8_DEG,
+        ],
+    )
+    def test_weights(self, weights: NeuralGCM_Weights) -> None:
+        pytest.importorskip('neuralgcm', minversion='0.2.0')
```

Comment:
You can add this to the top of the script below the imports instead of repeating it in every method.